packages/functional-tests/README.md

```diff
@@ -198,28 +198,28 @@ async signOut() {
 }
 ```
 
-6. Use `beforeEach` and `afterEach`: Use `beforeEach` and `afterEach` hooks to set up and tear down the test environment, or using `test.slow()` to mark the test as slow and tripling the test timeout, or running a test in a particular environment etc.
+6. Use `fixtures` to set up and tear down the test environment or run a test in a particular environment etc.
 
-7. When writing any test that use Firefox Sync, use the `newPagesForSync` helper function. This function creates a new browser and a new browser context to avoid any Sync data being shared between tests. After your test is complete, ensure that the browser is closed to free up memory.
+7. Use `test.slow()` to mark the test as slow and triple the test timeout.
+
+8. When writing tests that use Firefox Sync, use the `newPagesForSync` helper function. This function creates a new browser and a new browser context to avoid any Sync data being shared between tests. After your test is complete, ensure that the browser is closed to free up memory.
```

Comment:
Should we instead recommend using the `syncBrowserPages` fixture? Usage in tests looks a bit different that the description and example here.

Reply:
Yes! That would be much better! I squashed in the change.